doc/man3/X509_dup.pod

```diff
@@ -359,6 +359,15 @@ algorithms from providers. This created object can then be used when loading
 binary data using B<d2i_I<TYPE>>().
 
 B<I<TYPE>_dup>() copies an existing object, leaving it untouched.
+Note, however, that the internal representation of the object
+may contain besides the ASN.1 structure further data, which is not copied.
```

Comment:
This might read better the other way round:

```
Note, however, that the internal representation of the object
may contain further data in addition to the ASN.1 structure, which is not copied.
```

Reply:
How if instead putting the "besides the ASN.1 structure" in parentheses?

Reply:
That works, yes

Reply:
done